src/components/container/ConnectDevicesMenu/ConnectDevicesMenu.previewdata.ts

```diff
@@ -82,4 +82,34 @@ export const previewHealthConnectStatus: HealthConnectStatus = {
     enabledQueryRules: [],
     running: false,
     lastPrompted: ""
+};
+
+export const sampleParticipantInfo : ParticipantInfo = {
```

Comment:
Not a big deal, but do we need to fully populate this?  You can probably just use a few `as Type` assertions and only populate the fields you need for this component to render.